app/models/user.rb

```diff
@@ -614,11 +604,7 @@ def banned_at_date?(date)
   end
 
   private def senior_delegate_roles
-    active_roles.select do |role|
-      UserRole.is_group_type?(role, UserGroup.group_types[:delegate_regions]) &&
-        UserRole.is_lead?(role) &&
-        UserRole.status(role) == RolesMetadataDelegateRegions.statuses[:senior_delegate]
-    end
+    delegate_roles.select { |role| role.metadata.status == RolesMetadataDelegateRegions.statuses[:senior_delegate] }
```

Comment:
Here you dropped the `is_lead?` check from the previous implementation. Are you sure this is risk-free?

Reply:
Yes it is, because senior delegate is anyway a lead.